script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -126,7 +126,7 @@ I could easily jump over it, but...
 // Vampire Havana syndrome
 Probably something to do with sonic interference.
  Even if it isn't as bad as her Mystic Eyes,
- I'm sure it'd still be enough to scramble my thoughts and knock me off-balance.
+ I'm sure it'd still be enough to scramble my thoughts and knock me off balance.
```

Comment:
Did this one come up earlier? I might keep the -

Reply:
Yes it did thrice in fact.